packages/app/app/containers/ShortcutsContainer/index.js

```diff
@@ -149,7 +150,19 @@ class Shortcuts extends React.Component {
     ]);
   }
 
-  shouldComponentUpdate() {
+  componentWillUpdate() {
+
+  }
+
+  shouldComponentUpdate(nextProps) {
```

Comment:
Shouldn't this be in `componentDidUpdate`?

Reply:
I changed it